deploy/crds/operator_v1_authentication_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: operator.tigera.io/v1
+kind: Authentication
+metadata:
+  name: tigera-secure
+spec: {}
```

Comment:
I don't know what this file is used for exactly, so maybe it's fine that it omits some required values.

Reply:
It's just some dummy code generation, it's fine that it's empty